packages/oci_cli.rb

```diff
@@ -13,14 +13,14 @@ class Oci_cli < Package
   binary_url({
     aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/oci_cli/2.24.3_armv7l/oci_cli-2.24.3-chromeos-armv7l.tpxz',
      armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/oci_cli/2.24.3_armv7l/oci_cli-2.24.3-chromeos-armv7l.tpxz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/oci_cli/2.24.3_i686/oci_cli-2.24.3-chromeos-i686.tpxz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/oci_cli/2.24.3_x86_64/oci_cli-2.24.3-chromeos-x86_64.tpxz'
+    i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/oci_cli/3.1.2_i686/oci_cli-3.1.2-chromeos-i686.tpxz',
+  x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/oci_cli/3.1.2_x86_64/oci_cli-3.1.2-chromeos-x86_64.tpxz'
```

Comment:
The arm binary version doesn't match the package version?

Reply:
Yup. Rebuilding this one on arm...

Reply:
done